Sources/Subs-Attachments.php

```diff
@@ -661,6 +661,15 @@ function createAttachment(&$attachmentOptions)
 	$size = @getimagesize($attachmentOptions['tmp_name']);
 	list ($attachmentOptions['width'], $attachmentOptions['height']) = $size;
 
+	if (($exif_data = @exif_read_data($attachmentOptions['tmp_name'])) !== false && !empty($exif_data['Orientation']))
```

Comment:
In case the EXIF extension isn't loaded:
```suggestion
	if (function_exists('exif_read_data') && ($exif_data = @exif_read_data($attachmentOptions['tmp_name'])) !== false && !empty($exif_data['Orientation']))
```

Reply:
The one in Subs-Graphics.php would require the same change then

Reply:
Done